contracts/src/v0.8/ccip/test/DonIDClaimer.t.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {Ownable2Step} from "../../shared/access/Ownable2Step.sol";
+import {DonIDClaimer} from "../DonIDClaimer.sol";
+import {Test} from "forge-std/Test.sol";
+
+contract MockCapabilitiesRegistry {
```

Comment:
Please move to separate file


Also note the test structure of all other tests and follow that. We use a file per function to be tested.